examples/discovery-search-app/cypress/integration/passages/spec.ts

```diff
@@ -90,5 +90,27 @@ describe('Passage Results', () => {
       cy.get('@combinedResult').contains('View passage in document');
       cy.get('@combinedResult').contains('View table in document');
     });
+
+    describe('and clicking on "CI Document" for a result', () => {
+      beforeEach(() => {
+        cy.get('button[data-testid="search-result-element-preview-button"]')
+          .contains('View table in document')
+          .click();
+        cy.findByText('CI');
+        cy.findByText('CI').click();
+      });
+
+      it('verify that CI Document contains filters', () => {
```

Comment:
These messages should describe what you expect the application to do, not what the test should do (if that makes sense). (e.g. 'displays the filters')

Reply:
make sense